.github/workflows/ci-cd.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       release-hash: ${{ steps.release-data.outputs.hash }}
 
     steps:
-      - name: Generate token for Github PR Bot
+      - name: Generate token for GitHub PR Bot
```

Comment:
Should [this](https://github.com/modal-labs/modal-client/pull/3200/commits/e97158c3b5f22fac44f15689b4919e612191c09d) rather be a separate PR? 😃